app/src/ui/diff/text-diff.tsx

```diff
@@ -187,7 +187,7 @@ function showSearch(cm: Editor) {
   ) {
     searchLabel.style.display = 'none'
     searchField.placeholder = 'Search'
-    searchField.style.width = null
+    searchField.style.width = null!
```

Comment:
@rafeca I kept the behavior as it was, but this seemed like another place you might want to make a more positive change.

Reply:
I saw that one as well but honestly I don't know what can we replace it with 😅 

I'm in favor of leaving it like this for now 🙃